loco-new/src/generator/executer/filesystem.rs

```diff
@@ -88,7 +94,7 @@ impl Executer for FileSystem {
     fn create_file(&self, path: &Path, content: String) -> super::Result<PathBuf> {
         let target_path = self.target_dir.join(path);
         if let Some(parent) = path.parent() {
-            fs_extra::dir::create_all(parent, false)?;
+            std::fs::create_dir_all(self.target_dir.join(parent))?;
```

Comment:
please use `fs_extra`

Reply:
Done. I asked about this change in #1127 and assumed it was OK from the response.